src/api/todos.ts

```diff
@@ -0,0 +1,18 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const getTodos = (userId: number) => {
```

Comment:
It seems you've implemented the basic CRUD operations for the todos, but you're missing the 'toggle' and 'rename' functionality mentioned in the task description. You should add functions for these features.